WordPress/gutenberg
# editPost: deprecate __experimentalPluginPostExcerpt

## What?


It deprecates `__experimentalPluginPostExcerpt` component in the `editPost` package, ~and exposes the `PluginPostExcerpt` component in the `editor` package.~
Follow up of https://github.com/WordPress/gutenberg/pull/57096
Addressing [this feedback](https://github.com/WordPress/gutenberg/pull/55200#issuecomment-1857428093).

## Why?


Because we stopped using these prefixes.

## How?


* Deprecate `wp.editPost.__experimentalPluginPostExcerpt` from 6.6.
* ~Introduce `wp.editor.PluginPostExcerpt` from 6.6~

## Testing Instructions





* Compile the app.
* Edit/create a new post
* Open the post sidebar
* Open the panel to add/edit post excerpt
* Confirm the panel keep working as expected

<img width="1290" alt="image" src="https://github.com/WordPress/gutenberg/assets/77539/b7294dbb-d174-41a0-8286-363ba36cd460">


### Check component functions

* Open the dev console
* Confirm  `wp.editPost.__experimentalPluginPostExcerpt` is defined
* Confirm `wp.editPost.__experimentalPluginPostExcerpt` is deprecated


### Testing Instructions for Keyboard


## Screenshots or screencast
